crates/daphne-server/src/router/extractor.rs

```diff
@@ -177,7 +175,7 @@ where
 
         // TODO(mendess): this allocates needlessly, if prio supported some kind of
         // AsyncParameterizedDecode we could avoid this allocation
-        let payload = hyper::body::to_bytes(body).await;
+        let payload = axum::body::to_bytes(body, usize::MAX).await;
```

Comment:
we may want to only accept a certain amount of bytes based on the payload we are expecting. currently this is not supported, right?

Reply:
Yes, I'm gonna open an issue. The previous implementation didn't allow limiting this so I kept it "the same" but we should investigate if we can deduce a sane limit we can put here